### DISCUSSION TITLE
Is it possible to exclude a display from the "next/previous display" keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by gsinti** [rxhanson/Rectangle#1318] [Q&A]
My typical workflow is to use my MacBook with a single external monitor. However, I occasionally use my iPad as a third display for some spillover.

I rely on the next/previous display hotkey often to move windows between my built-in display and external monitor, but find it distracting when I need to repeat the hotkey an additional time as the window sometimes lands on my iPad unintentionally.

Is there a way to exclude the iPad display from this hotkey altogether? Or some method to only have the hotkey work for Display 1 and Display 2?

EDIT: Alternatively, is there a way to target a specific display when sending a window to another display?

**Comment by rxhanson** [comment]
Currently there is not a way to do this built into the app. I am not opposed to a good pull request that would add this in, but I don't consider it a commonly desired feature and would only consider a pull request that doesn't add it to the UI.

> Alternatively, is there a way to target a specific display when sending a window to another display?

Not in Rectangle. In [Rectangle Pro](https://rectangleapp.com/pro) (paid), you can create a "Custom Size & Position" and specify a display.
